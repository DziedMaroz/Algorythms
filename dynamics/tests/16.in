302063801695224
726
616767797575
007717
2
485
617917
1236
25
937
81836009
143171058167
666348
2907954893685
29723410
4922932192
94
972631
9763
54
908746

39665122589
0566
1320413524
74236281

40155
951364
15041071868
908
68300049227525
440
4253098450
8323779799017
0815378
5
72
84
755718
55600441783144
035422933267
11082923221
2875
831193
78939439509
10530
0
7473031
50458
091250499
732029993407
880015
8486
68116831201967
532
07470752110
655427453393
127
03786142143407
047298573
36084
158
86722
3596095909424
54453050
05769
46911107
5553
4965911690726
5809258515
23287994
3217142090403
6352

747617472
3408071113

56670445844216
076075
62221304255
9814090241012
00028

1
275031172
88225
0

5571
99324248896
19
7524425015189
99748013063899
8978123214
53435288131216
03558
2795990709
889148210653
994
5372835961764
019917706897
2
0
4889551380
7539277775348

7676696
6039516508
2933400
3264514957146
4961275598
600091
2754140381
24914329537
5378
69545944582
88
4325792453260
12923995
7016167
2267
617
1
8316648
29378613580
026713
23718594
984906550
4
6
932377351
17
7

802341857
675982951
652673722444
47222
368
4593243340
7765976917150
00751444

9150780
5461
27497095797

412453343143
93080239059
7
54849261966
3469832
536857333393
4
0886166
8013396748934
14189321854822
0479
2
3911508124
959578073456
80977377320

491
78
71
11
3
2548
7
26
5005
370577354
7815381
6842248549171
27296759873
31276338189
8
0815147
407
9
42167503
22864
61070421604
92
16864
7811882468570
40536
1461284
59340382228277

7778203807601
385130
03
155
801
01197334
966152717140
0818871

433405001172
38
23
4
59584839
8599762683
9258
5497876172513
36
3802

25
5182337
28991555
7248
0575871851115
05007
327842
569380382744
512939
71890
5
235463007059
24392
2602526388
56644849783
680
2725033557
3081939

38327773630

0689949

30733463
02856024646
7418154
8618429785
9846
760849
6
547558130
18519294547
807265619598
91271226878
57389
842686
501881597892
297255
98724913
6474380617
5116712
11
25588
6846841
3939
806
362498904431

58
71384144
6

77383611151128
196615008
068558024
45
5951668

4
62704274
08797996
5872115659
5920994
7659254
944065227126
918
9811
987
29022
17582
0174
953
03041737
3330334
78926751
00838904773874
02190127521
30137495756929
12994
08747059
2
1732723342
5451
758280346
2084544946147
155492
079053068
2332159436
19083297216
566472
808555551
992541
5834696114
6049658427
48984305859
734205691850
2893121029584
3244050

0195370358783
873622148
6
4098061969
7725127165820
9847
6671
74102915

5
986338650
3110
4
312208002387
2
06920297
2316810
07805257
8
321380824
7022
97680433927

9233
41860398323611
14878656000057
3
1389448308
2053238879
9
16159
1522601
5104815023
6853213121
909
501210335699
768368538
7482462761886

40020050
62695
06743624937
1
5
038
6620515870
821
2930618836153
60755620870
054362
27291

6925068159
15725

2717264
04895
5089
259521178715
445217167876
005260
60016488
14092276624

08
264027332010
49566344929159
372669295183
05
7245514
9684002887427
730
83636476940092

671281
0046115
250541644883
24357324031
41532969170
30
22978
67791791733121
6
75768568595183
99920
0434
8191288158921
3911921
6204958
128517044
624061176486
454976850
641009025723
57710210
798270395
65394504871611
42716062
50414940
68376010988728

4
618

4225
05
7
3376795801
921915
5311
3903825
6242509
92430098833142
14
5206092
82468
8
195807717219
0113
2418258492833
90238
86821543
3633702692070
839
4016863800743
387299210
1043659
0417425
8169623200
1431706684961
432312440641
66
19684
8956258521
104729050620

18421146016215
325375
0664694521
58220227506546
4712398949824
485587751
171952
56072
964431708866
602
9
3568664484489
545674726660
54442948
12404402
014035416905
3084304
100186
9630083
130
656138987
976466326228
062406141
59922834074
01306857
43843628
2757
350252

5
8
5089470905
38555
4202354
3680507584919
036
59745
254798161354
256434
979246
8505412759780
86
508
8
766831417091

64882925401195
78544

4054438917965
6855
640
4214559171
2597198572
3989161
029
6439565165
26363763576680
4947
7
28814442042932
98262
3
4
95
29
903399
711393243
492348356620
93
77992476237
346
48
191
60716739
305281126164
069271848
7405990332028
7459824863
499401668
4521890110584
4
65022498
29

69230471832
813211073204
79844820108
046
76895
5092721861
8844387254
38
575084377
4792050370151
5823
853
67755433262
8
9292747
15198464691826
9491753177
48071819133
20
376141761475
21
9460998588
4985091886701
44996167999
834056570013
188252
37686183450
878
324022857
634733
2841883183
9406614776588
87959775
420696
81088
089098398
2338
23
1129662518
66306
544703
19452
58667902426581
63412646710110
3
341035565321
24887160240
500830
28885830252
1621451

5982405
152
36677172233
0863
02513362150
4941411159968
301

689845766451
55
702154701
5568305891106

83570
6633
7969302
56
022
52821938638
026465000
359451066909
63036004
4360
47781998839
308549
90030

46
74
1592495
1
5652787586
1482484505
03489537458
221
326

7
5253298362
150849033299
744325
692656
344439079880
5071383
259922
533
28111
12
2698635
7
8654018426
47074811
31930672038547
367018241
284656021
496562

071
046426684313
5844730603
631281
26
37
24545595
9830265073748
546180
67946435609527
721323554668
923364797286
211
71126
36
70003272333
3328518
34
140749
17268390263
081
6981
1609371765830
29329289194
56175523686987
7
1974120486
21264200771176
3486

9543271758733
97894
33844483
0680
29220603962898
19
39791
7
8271857219048
494875911
22184716716490
366
579
7704637

0487349167083
249988819540
59605269
226
01
7387621324002
46024629847995
556699
995310900251
282207619
2992
811176
997178
9289159
4
38567612566398
8199216873148
369
9
442957934
1937
75718935790790
731858329025
144236018
8490
771991389
27828110
4316823817
67243453571553
800980
6
2991929724946
61560
04156644503
81404372734
84334408672047
52828
9040
9864
76995672257374
376
5
92177867
662393026
56225457
