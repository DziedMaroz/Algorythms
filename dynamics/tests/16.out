10
30 20 6 3 801 6 95 2 2 4
